### DISCUSSION TITLE
Any latest next.js 9.3 setup repo link with redux and redux persist library?

### DISCUSSION THREAD

**Discussion by iamfiropo** [vercel/next.js#12709] [Help]
Just starting out on building a production ready next.js app and i'd love to have a link to an up to date repo on next.js 9.3 setup using redux and reselect libraries.

**Comment by mvasilkov** [comment]
This is my up-to-date template: https://github.com/mvasilkov/next-redux (see the example folder).

I keep it really minimalistic, with as little dependencies as possible.

No reselect, but should be trivial to add.

**Reply by iamfiropo** [reply]
Thanks @mvasilkov 

Hoping to get a redux with redux persist latest setup
